backend/src/test/factory/DiscountFactory.ts

```diff
@@ -0,0 +1,11 @@
+import { DiscountType } from '../../main/data/DiscountType';
+import { Discount } from '../../main/entity/Discount';
+
+export class DiscountFactory {
+  generate(discountType?: DiscountType, discountValue?: number): Discount {
+    return new Discount(
+      discountType ?? DiscountType.AMOUNT,
```

Comment:
What does the `??` notation mean? Also wondering why we set it to 188.08 for discountValue?